Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM rust:1.61 as build
+FROM rust:1.67 as build
```

Comment:
@hugotiburtino Shall we add a .tool-versions file in this repo and make all changes with a pinned `rust` version?!

Reply:
It sounds safer.
Since Rust is still envolving, I would just say that we should keep upgrading its version, in order to get the benefits of new language features (v.g. the last thing, that we got to know, was string interpolation). 
If we don't want to have the extra work of keeping upgrading it, I would say let's leave as it is.